.github/workflows/linux-cpu-x64-nightly-examples.yml

```diff
@@ -0,0 +1,65 @@
+name: "Linux CPU x64 Nightly Examples"
+
+# Run at 5:00 AM UTC every day
+# 9:00 PM PST
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 5 * * *"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+env:
+  ort_dir: "onnxruntime-linux-x64-1.18.0"
+  ort_zip: "onnxruntime-linux-x64-1.18.0.tgz"
+  ort_url: "https://github.com/microsoft/onnxruntime/releases/download/v1.18.0/onnxruntime-linux-x64-1.18.0.tgz"
+jobs:
+  Nightly_test_examples:
+    runs-on: [ "self-hosted", "1ES.Pool=onnxruntime-genai-Ubuntu2204-AMD-CPU" ]
+    steps:
+      - name: Checkout OnnxRuntime GenAI repo
+        uses: actions/checkout@v2
+
+      - name: Download OnnxRuntime
+        run: |
+          curl -L -o ${{ env.ort_zip }} ${{ env.ort_url }} 
+
+      - name: Unzip OnnxRuntime
+        run: |
+          tar -xzf ${{ env.ort_zip }}
+          rm ${{ env.ort_zip }}
+
+      - name: Rename OnnxRuntime to ort
+        run: |
+          mv ${{ env.ort_dir }} ort
```

Comment:
We do not need these steps anymore. ort-genai will auto download the ort dependency at build time.

Reply:
Okay, removed this from our nightly pipeline as well.